sympy/simplify/simplify.py

```diff
@@ -2703,9 +2703,13 @@ def update(b):
         bases = []
         for b, e in c_powers:
             b, e = bkey(b, e)
-            common_b[b] = e
+            if b in common_b.keys():
+                common_b[b] = common_b[b]+e
```

Comment:
Looks good so far!  Please 1) add space around `+` and then 2) add a test from the issue to the test_powsimp suite (labeling it with the issue as "issue ####").